packages/widgetbook/lib/src/fields/color_field/hsl_color_text_fields.dart

```diff
@@ -0,0 +1,120 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+import 'color_text_field.dart';
+
+class HslColorTextFields extends StatelessWidget {
```

Comment:
The comments here will apply to all other similar widgets as well (i.e. RGB and HEX widgets).

1. Should be renamed to `HslColorPicker`.
1. Should be converted to `StatefulWidget`, since `colorValue` is being manipulated.
1. `onChanged` should send out `Color` and not `List<String>`.
1. No need to assign the `key` of the `TextField`s.


You can have a look at [this widget](https://github.com/YoussefRaafatNasry/color_picker/blob/main/lib/src/pickers/hsl_color_picker.dart) for more info.